src/http.rs

```diff
@@ -53,38 +59,53 @@ pub async fn do_http_request(
         add_metric!(request_cycles_charged, cost);
         add_metric!(request_cycles_refunded, cycles_available - cost);
     }
-    inc_metric_entry!(host_requests, host);
-    let request_headers = vec![
+    inc_metric_entry!(host_requests, host.to_string());
+    let mut request_headers = vec![
         HttpHeader {
             name: "Content-Type".to_string(),
             value: "application/json".to_string(),
         },
-        HttpHeader {
-            name: "Host".to_string(),
-            value: host.to_string(),
-        },
+        // HttpHeader {
```

Comment:
dead code?

Reply:
This header is a discrepancy between the Phase 1 Ethereum canister and ckETH implementations. I'm currently unsure if there was a reason that this was originally included in the Phase 1 canister, so I've made a note for the next weekly sync in case anyone has any insight about this.